src/bootstrap/bootstrap.py

```diff
@@ -734,7 +734,7 @@ def build_bootstrap(self, color):
             if "LIBRARY_PATH" in env else ""
 
         # Export Stage0 snapshot compiler related env variables
-        build_section = "target.{}".format(self.build)
+        build_section = "target.'{}'".format(self.build)
```

Comment:
This isn't the correct fix, this will break ,`target.x86_64-unknown-linux-gnu` (without the quotes). Please add that as a test case and fix this in `get_toml`, not at the call site.

Reply:
As requested, I attempted the fix in `get_toml`.
Thank you for your patience.